### ISSUE TITLE
Remove unnecessary icons

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [tengusw/share_to_clipboard#48]
After I download the package from F-Droid, opened with [X-plore](https://play.google.com/store/apps/details?id=com.lonelycatgames.Xplore) (long press the package icon and click Open as ZIP), I saw there are numerous icons in the package (for example, drawable-hdpi-v4), but they haven’t been used, is it possible to exclude these icons into the package?
![Screenshot_20210315-165250](https://user-images.githubusercontent.com/10364351/111135051-2f0fb500-85b7-11eb-90de-ab86f9bfe3d6.png)

**Comment by tasn** [comment]
I think they should be used on older devices that don't have these built-in by the OS, though I'm not sure tbh. What's the total wasted size? I'd expect it to be tiny anyway so not worth the effort of even figuring this out.

**Comment by KrasnayaPloshchad** [comment]
> What's the total wasted size? I'd expect it to be tiny anyway so not worth the effort of even figuring this out.

Yeah, it would be helpful if there is a utility like [AppResourcesUsageView](https://www.nirsoft.net/utils/app_resources_usage_view.html) for Android developers.
